.github/workflows/publish_to_wiki.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish to Wiki
+
+# Trigger this action only if there are changes pushed to the docs/** directory under the main branch
+on:
+  push:
+    paths:
+      - markdown/wiki/** # This includes all sub folders
+    branches:
+      - main # This can be changed to any branch of your preference
+
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }} # This is the repository secret
+  USER_NAME: <USERNAME> # Enter the username of your (bot) account
+  USER_EMAIL: <EMAIL> # Enter the e-mail of your (bot) account
+  OWNER: ${{ github.event.repository.owner.name }} # This is the repository owner
+  REPOSITORY_NAME: ${{ github.event.repository.name }} # This is the repository name
+
+jobs:
+  publish_to_wiki:
+    name: Publish to Wiki
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install podman
+        run: |
+          apt-get -qq -y podman
+      # 1. Create folder named `tmp_wiki`
+      # 2. Initialize Git
+      # 3. Pull old Wiki content
+      - name: Pull content from wiki
+        run: |
+          mkdir tmp_wiki
+          cd tmp_wiki
+          git init
+          git config user.name $USER_NAME
+          git config user.email $USER_EMAIL
+          git pull https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git
+      # 4. Synchronize differences between `docs` & `tmp_wiki`
+      # 5. Push new Wiki content
+      - name: Push content to wiki
+        run: |
+          rsync -av --delete markdown/wiki/ tmp_wiki/ --exclude .git
+          cd tmp_wiki
+          for i in *.template.md; do
+            f="${i%.template.md}.md"
+            make "${f}"
+            git add "$f"
+          done
+          git add *.png
+          git commit -m "Update Wiki content"
+          git push -f --set-upstream https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git master
```

Comment:
How about using https://github.com/marketplace/actions/update-wiki ?

Internally [it's just a simple bash-script](https://github.com/orlovm/Wiki-Action/blob/master/update-wiki.sh), which is easily audited :grin: